docs/guides/01-getting-started.md

```diff
@@ -100,43 +103,43 @@ carto.setDefaultAuth({
 });
 ```
 
-### Define source
+### Create source
 
-The next step is to define the [`source`](/developers/carto-vl/guides/02-using-sources) from your account to be displayed on the map. In the example below, the `source` is a dataset named `populated_places` with all the populated places around the world from [Natural Earth](https://www.naturalearthdata.com/).
+The next step is to create the [`source`](/developers/carto-vl/guides/using-sources/) from your account to be displayed on the map. In the example below, the `source` is a dataset named `populated_places` with all the populated places around the world from [Natural Earth](https://www.naturalearthdata.com/).
```

Comment:
I'm afraid we are working in parallel fixing the same things... 😞

Reply:
:(